setup.py

```diff
@@ -1,36 +1,7 @@
-import os
+import pathlib
 import setuptools
 
 
-def read(fname):
-    with open(os.path.join(os.path.dirname(__file__), fname)) as file:
-        return file.read()
-
-
 setuptools.setup(
-    name='epcsunspecdemo',
-    long_description=read('README.rst'),
-    long_description_content_type='text/x-rst',
-    packages=setuptools.find_packages('src'),
-    package_dir={'': 'src'},
-    include_package_data=True,
-    install_requires=[
-        'attrs',
-        'click',
-        'pysunspec',
-        'toolz',
-        'tqdm',
-    ],
-    extras_require={
-        'development': [
-            'gitignoreio',
-        ],
-    },
-    entry_points={
-        'console_scripts': [
-            (
-                'epcsunspecdemo = epcsunspecdemo.cli:cli',
-            ),
-        ],
-    },
+    long_description=pathlib.Path("README.rst").read_text(encoding="utf8"),
 )
```

Comment:
Can be moved over with another special bit.

https://github.com/altendky/qtrio/blob/f8ef0a86d41b15f0b189edf888a204b7c9d6d35a/setup.cfg#L18
```ini
long_description = file: README.rst
```

(I would include a GitHub suggestion/patch to ease quick application of the removal here...  but you apparently can't do that when including deleted lines.  oh well.)